simulation/random_topology_generator.py

```diff
@@ -26,79 +31,86 @@ def random_connected_graph(v, e):
     tree = permute(tree)
 
     for i in range(1, v):
-        j = ran( i )
-        adj_matrix[ tree[ i ] * v + tree[ j ] ] = 1
-        adj_matrix[ tree[ j ] * v + tree[ i ] ] = 1
+        j = ran(i)
+        adj_matrix[tree[i] * v + tree[j]] = 1
+        adj_matrix[tree[j] * v + tree[i]] = 1
 
     count = v - 1
-    max_edge = v * (v - 1)/2
+    max_edge = v * (v - 1) / 2
     e = max_edge if e > max_edge else e
     while count < e:
-        i = ran( v )
-        j = ran( v )
+        i = ran(v)
+        j = ran(v)
 
         if i == j:
             continue
 
-        if i > j :
+        if i > j:
             i, j = j, i
 
         index = i * v + j
-        if not adj_matrix[ index ]:
-            adj_matrix[ index ] = 1
+        if not adj_matrix[index]:
+            adj_matrix[index] = 1
             count += 1
 
     return adj_matrix
 
+
 def permute(arr):
     return random_permutation(arr)
 
+
 def random_permutation(iterable, r=None):
     "Random selection from itertools.permutations(iterable, r)"
     pool = tuple(iterable)
     r = len(pool) if r is None else r
     return tuple(random.sample(pool, r))
 
+
 def init_array(arr, end):
-   for i in range(0, end):
-      arr[i] = i
+    for i in range(0, end):
```

Comment:
why one more indentation?

Reply:
That was inconsistent with the rest of the indentation in the file. (This one was 3 white space long while others are 4 white space long)